### ISSUE TITLE
[Footer] Color specs update

### ISSUE THREAD

**Issue by aweell** [dxc-technology/halstack-style-guide#299]
## Story description

- [ ] Update component documentation page with corrected color specs

## React

```
    backgroundColor: globalTokens.black,
    bottomLinksDividerColor: globalTokens.mediumBlue,
    bottomLinksFontColor: "",
    copyrightFontColor: "",
    customContentFontColor: "",
    fontColorBase: globalTokens.white,
```

## Angular **(Pending update)**

```
  "--footer-backgroundColor": globalTokens.black,
  "--footer-fontColor": globalTokens.white,
  "--footer-lineColor": globalTokens.mediumBlue,
```

## Footer color specs table

|   Component token                         |   Element                     |   Core token                            |   Value             |
| --------------------------------- | --------------------- | ---------------------------- | -------------- |
|   `backgroundColor`                      |   Footer container       |   `Hal-Black`                          |   `#000000`   |
|   `bottomLinksDividerColor`         |   Divider                       |   `Hal-Blue-L-50`                  |   `#0095ff`      |
|   `bottomLinksFontColor`             |   Bottom links               |   `Hal-White`                         |   `#ffffff`         |
|   `copyrightFontColor`                  |   Copyright                   |   `Hal-White`                         |   `#ffffff`         |
|   `socialLinksColor`                       |   Social icons                |   `Hal-White`                         |   `#ffffff`         |

**Comment by GomezIvann** [comment]
Changes made on the Footer advanced tokens:

- [ ] `fontColorBase` token removed.
- [ ] `customContentFontFamily` token removed.
- [ ] `customContentFontSize` token removed
- [ ] `customContentFontStyle` token removed.
- [ ] `customContentFontWeight` token removed.
- [ ] `customContentFontColor` token removed.
- [ ] `bottomLinksFontColor` has a new value: `#ffffff`.
- [ ] `copyrightFontColor` has a new value: `#ffffff`.
- [ ] `logoWidth` has a new value: `48px`.
- [ ] `minHeight` renamed as `height`.
- [ ] `socialIconSize` renamed as `socialLinksSize`.
- [ ] `socialIconsGutter` renamed as `socialLinksGutter`.
- [ ] `socialLinksColor` token added.

**Comment by aweell** [comment]
Closed via #353
